samples/javascript/chatapp/react/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Create a Chat app with client SDK and react
+
+## Prerequisites
+
+1. [Node.js](https://nodejs.org)
+2. Create an Azure Web PubSub resource
+
+## Overview
+The sample demonstrates building a chat app on a webpage with the Web PubSub client SDK and react.
```

Comment:
we could also mention that this sample is modified on the hello-world react project, which explains why it contains some not used css styles and files